### PR TITLE
Added favicon

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Added favicon to website.

Issue #75

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +15,7 @@
     <link rel="stylesheet" href="stylesheets/contributors.css">
 
     <link href="https://fonts.googleapis.com/css?family=Raleway|Lato:400,900|Quicksand|Open+Sans|Roboto+Mono" rel="stylesheet">
-    <!--[if lt IE 9]>
-        <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="icon" type="image/x-icon" href="/images/logos/jboss-logo.ico"/>
```
Comment (aashutoshrathi) [inline index.html]: You confused me for a while 😆.
Change ` <link rel="icon" type="image/x-icon" href="/images/logos/jboss-logo.ico"/>` to ` <link rel="icon" type="image/x-icon" href="/images/logos/jboss-logo.ico">`, deployment is not showing favicon

Comment (SachinRammoorthy) [reply index.html]: Works for me 🤔 
screen shot 2017-12-31 at 10 32 10 pm

Comment (SachinRammoorthy) [reply index.html]: Let me try committing again to trigger another deployment.

**Comment by SachinRammoorthy** [comment]
New commit to trigger automatic build

**Comment by aashutoshrathi** [comment]
Actually, don't squash for deploy to work

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi this link works (the second one): https://bot.jboss-outreach.org/2017-12-31/1bccf02/

File: index.html [hunk 2]
```diff
@@ -15,11 +15,9 @@
     <link rel="stylesheet" href="stylesheets/contributors.css">
 
     <link href="https://fonts.googleapis.com/css?family=Raleway|Lato:400,900|Quicksand|Open+Sans|Roboto+Mono" rel="stylesheet">
-    <!--[if lt IE 9]>
-        <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="icon" type="image/x-icon" href="./images/logos/jboss-logo.ico"/>
```
Comment (aashutoshrathi) [inline index.html]: You need to remove `/` anyway. 😄

File: index.html [hunk 3]
```diff
@@ -15,11 +15,9 @@
...
 
-    <title>GCI | JBoss Community</title>
+    <title>GCI | JBoss Community</title>
```
Comment (aashutoshrathi) [inline index.html]: And extra space can be removed now.

File: index.html (hunk 2 above)
Comment (SachinRammoorthy) [reply index.html]: Should I squash the commit?

Comment (aashutoshrathi) [reply index.html]: Yes go on

Comment (SachinRammoorthy) [reply index.html]: ?

Comment (aashutoshrathi) [reply index.html]: Yes, squash after doing requested changes

File: index.html [hunk 4]
```diff
@@ -15,9 +15,7 @@
     <link rel="stylesheet" href="stylesheets/contributors.css">
 
     <link href="https://fonts.googleapis.com/css?family=Raleway|Lato:400,900|Quicksand|Open+Sans|Roboto+Mono" rel="stylesheet">
-    <!--[if lt IE 9]>
-        <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
+    <link rel="icon" type="image/x-icon" href="images/logos/jboss-logo.ico"/>
```
Comment (aashutoshrathi) [inline index.html]: 😆 not that one, the one before `>`.
make it `<link rel="icon" type="image/x-icon" href="./images/logos/jboss-logo.ico">`

Comment (SachinRammoorthy) [reply index.html]: oh 😆

Comment (SachinRammoorthy) [reply index.html]: done

**Comment by aashutoshrathi** [comment]
Always write `Fixes #issue-number` at top of PR message

**Pull request merged**